.gitlab-ci.yml

```diff
@@ -0,0 +1,144 @@
+variables:
+  DEBIAN_FRONTEND: noninteractive
+  DOCKER_DRIVER: overlay2
+  ARTIFACTS_DIR: "$CI_PROJECT_DIR/dist"
+  # Set up variables for pushing the generated tags
+  DEPLOY_GIT_REPO: "git.cells.es/$CI_PROJECT_PATH.git"
```

Comment:
In this push I would eliminate ALBA-specific hardcodings.
Alternatively, I would callthis file `.gitlab-ci-alba` to avoid it automatically triggering pipelines that won't work if someone else pushes it to their own gitlab.

Reply:
The easiest is to simply comment that the `ARTIFACTS_DIR` variable should be set in the gitlab project settings